setup.py

```diff
@@ -57,7 +57,7 @@ def load_requirements(*requirements_paths):
     packages=find_packages(include=['openassessment*'], exclude=['*.test', '*.tests']),
     include_package_data=True,
     # Todo - this should be loading 'requirements/base.in' but Tox is having an issue with it
-    install_requires=load_requirements('requirements/base.txt'),
+    install_requires=load_requirements('requirements/base.in'),
     tests_require=load_requirements('requirements/test.txt'),
```

Comment:
Should probably also switch to `test.in` while you're at it.  And then I think you could drop `requirements/*.txt` from `MANIFEST.in`.

Reply:
@jinder1s I think if you add `requirements/*.in` to the MANIFEST.in file, tox won't balk at you switching this `install_requires` line to look at `base.in`.